src/Microsoft.DotNet.Tools.Builder/BuilderCommandApp.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Dnx.Runtime.Common.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.ProjectModel;
+using NuGet.Frameworks;
+using System.Linq;
+
+//This class is responsible with defining the arguments for the Compile verb.
+//It knows how to interpret them and set default values
+namespace Microsoft.DotNet.Tools.Compiler
+{
+    internal class BuilderCommandApp : CompilerCommandApp
```

Comment:
True. However, I need to somehow express the compiler command args in one place and then have the builder verb inherit those and add one extra flag. Without this class, all the args and their interpretation would be copy pasted between the two commands.

Do you have another implementation suggestion for expressing inheritable and extensible verb flags?